Code/Tools/FBuild/Documentation/docs/options.html

```diff
@@ -345,6 +349,15 @@ <h2>FBuild.exe Detailed</h2>
     <div class='newsitemheader' id="dist">-dist</div>
     <div class='newsitembody'>
 <p>Enable distributed compilation. Requires some build configuration.</p>
+
+    <div class='newsitemheader' id="distcompressionlevel">-distcompressionlevel [level]</div>
+    <div class='newsitembody'>
+<p>Control compression level of jobs sent out for distribution. (Default -1)</p>
+<p>This can be used to increase the level of compression for jobs sent out for distribution, trading increased CPU time
+        in order to reduce network transfer. This can be useful in network bandwidth limited environments. Note that
+        this value does not affect the compression level of the responses sent back from workers.</p>
+<p>See option <a href="#cachecompressionlevel">-cachecompressionlevel</a> for more details on compression levels and
+        their performance.</p>
```

Comment:
It's worth noting that that compression table performance is based on compression of .obj files and is very different (ratios and performance) to compression of preprocessed source code.

I think it'll be helpful to have a table here based on some preprocessed data. Tests exist in FASTBuild for this already so I can prepare that table from those.